language-server/src/unreal-buffers.ts

```diff
@@ -44,7 +44,7 @@ export enum MessageType
     FindAssets,
     DebugDatabaseSettings,
 
-    PingAlive,
```

Comment:
Why did you get rid of the PingAlive message here?

Reply:
It didn't seem like it was being used anywhere so I thought it was old, will add it back